libs/json-api-nestjs/src/lib/factory/ajv/utils/input-body-post-schema.ts

```diff
@@ -104,6 +102,14 @@ export function inputBodyPostSchema(
   };
 
   const relationships = Object.keys(relationsField).reduce((acum, item) => {
+    if (
```

Comment:
I think, better use json schema validate uuid.
You can use additional conditional before get from metadata information about ID.
like this:
```
if(isUuid){
relDataType.properties.data.properties.id.pattern = '^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$'
relDataType.properties.data.properties.id.minLength = 36
relDataType.properties.data.properties.id.maxLength = 36
}
```
so, before if, not need check in pipe